.github/actions/monitor-verified-batches/action.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: monitor-verified-batches
```

Comment:
nit: can we make name more descriptive ? we are publishing actions publicly, no? maybe: 
`monitor-cdk-verified-batches`

Reply:
updated in [chore: nit](https://github.com/0xPolygon/kurtosis-cdk/pull/131/commits/5ee1ee05002747a52da688bb2830d7849e799cf1)